src/dotnet/Chat.UI.Blazor/Components/Banners/TalkingMembersBanner.razor

```diff
@@ -0,0 +1,88 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@inherits ComputedStateComponent<TalkingMembersBanner.Model>
+@{
+    var m = State.Value;
+    if (ReferenceEquals(m, Model.None))
+        return;
+
+    if (!ScreenSize.IsNarrow())
```

Comment:
Better to put this check to Banners.razor and do not render TalkingMembersBanner on wide screens.